app/policies/roles/institution/health_organization.json.erb

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "name": "Institution <%= institution_name %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "institution/<%= institution_id %>",
+          "action": [
+            "institution:read",
+            "institution:update",
+            "institution:delete",
+            "institution:registerDevice",
+            "institution:createSite",
+            "institution:createEncounter",
```

Comment:
Note: this will conflict with encounter-site branch. the createEncounter is moved to site in that branch. Maybe removing is just enough sinse site resource is with `"*"` action. To be done on the latest branch to be merged for sure.